zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -804,9 +804,8 @@ pub(crate) fn get_wallet_transparent_output(
              OR (
                  (
                     t.mined_height IS NOT NULL -- tx is mined
-                    -- TODO: uncomment the following two lines in order to enable zero-conf spends
-                    -- OR t.expiry_height = 0 -- tx will not expire
-                    -- OR t.expiry_height >= :mempool_height -- tx has not yet expired
+                    OR t.expiry_height = 0 -- tx will not expire
+                    OR t.expiry_height >= :mempool_height -- tx has not yet expired
```

Comment:
So why was this commented out @nuttycom? I saw this during my confirmations work, but didn't point it out as it was an historical TODO.

Reply:
This was initially added commented-out in #1402 in what was called `wallet::transparent::get_unspent_transparent_output` (renamed in #1462).

That same PR also added a similar commented-out line in `get_spendable_transparent_outputs`, but that instance was removed by @nuttycom in 8f9efaf621c47d0841a2f07ef4051462ad720e7b as part of the confirmations policy PR #1888.